pkg/gce-cloud-provider/compute/gce.go

```diff
@@ -47,8 +48,33 @@ const (
 	regionURITemplate = "projects/%s/regions/%s"
 
 	replicaZoneURITemplateSingleZone = "projects/%s/zones/%s" // {gce.projectID}/zones/{disk.Zone}
+	versionV1                        = "v1"
+	versionBeta                      = "beta"
+	versionAlpha                     = "alpha"
+	googleEnv                        = "googleapis"
 )
 
+type versionMap struct {
+	versionV1    *compute.Service
+	versionBeta  *computebeta.Service
+	versionAlpha *computealpha.Service
+}
+
+var computeVersionMap = map[string]map[string]map[string]string{
+	googleEnv: {
```

Comment:
Will this map work under other env?

Reply:
Yes this will work with other environments too